parking_permits_app/models.py

```diff
@@ -0,0 +1,386 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.gis.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+# TODO: Some of these fields should come directly from Helsinki profile User-model.
```

Comment:
Is there already a ticket in Jira for this task so we don't forget it?

Reply:
@sam-hosseini Ticket added here: https://helsinkisolutionoffice.atlassian.net/browse/PV-164